src/request/components/request-detail-panel-request.tsx

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+/*tslint:disable:no-var-requires */
+const messageProcessor = require('../util/request-message-processor');
+/*tslint:enable:no-var-requires */
+
+import _ = require('lodash');
+import React = require('react');
+
+const getPayloads = (function() {
+    const getItem = messageProcessor.getTypePayloadItem;
+
+    const options = {
+        'web-response': getItem,
+        'web-request': getItem
+    };
+
+    return function(request) {
+        return messageProcessor.getTypeStucture(request, options);
+    };
+})();
+
+class RequestUrl extends React.Component<{url: string}, {}> {
```

Comment:
Does declaring the types like this buy us anything? Like it dosen't look like its a short cut for `this.props.url`... Does the type check come in the usage end with `<RequestUrl url={webRequestPayload.url} />`?